gems/pending/VMwareWebService/MiqPbmInventory.rb

```diff
@@ -0,0 +1,33 @@
+require 'VMwareWebService/PbmService'
+
+module MiqPbmInventory
+  def pbm_initialize(server, username, password)
+    @pbm = PbmService.new(server, username, password) if @apiVersion >= '5.5'
+  end
+
+  def pbmProfiles
```

Comment:
I was trying to stay consistent with MiqVimInventory which has (for example) `virtualMachines` which has a hash by key and `virtualMachinesByMor` which hash a hash by moref.  For PbmProfiles the key and ems_ref will be the same.